RecoLocalCalo/EcalRecProducers/python/ecalRecHit_cfi.py

```diff
@@ -49,7 +49,7 @@
                             
     # for channel recovery
     algoRecover = cms.string("EcalRecHitWorkerRecover"),
-    recoverEBIsolatedChannels = cms.bool(False),
+                            recoverEBIsolatedChannels = cms.bool(True),##default is false
```

Comment:
do I understand correctly that this means that this PR current state is still not final?
IIUC, the PR to be merged will have this flag set to False.

Reply:
@Slava77  @taroni as I already told you yesterday, yes, correct, that we still need to switch it off. And I said that we are waiting because we want to understand the chi2 issue. HOWEVER, as far seen, the chi2 issue is there wether or not the recovery is switched on. So we are not the culprit. Hence I would propose that we make the final commit switching off the recovery so that we can proceed with merging the PR. Do you agree ?